courses/beginner/10.md

```diff
@@ -12,30 +12,23 @@ In addition to its own controls, an InSpec profile can bring in the controls fro
 
 When a profile includes controls from another profile, it is usually referred to as a “meta profile” or a “profile overlay.” Those of us with Chef background sometimes call it a “wrapper profile”.
 
-::: info Further information on inheritance
-You can find out more about inhertiance here [https://www.inspec.io/docs/reference/profiles/#sts=Profile%20Dependencies](https://www.inspec.io/docs/reference/profiles/#sts=Profile%20Dependencies)
-:::
-
-### Defining the Profile Dependency  
-
-Before a profile (e.g. `Profile A`) can use controls from another profile (e.g. `Profile B`), `Profile B` needs to be included in `Profile A`'s `inspec.yml` file in the `depends` section. Each profile that `Profile A` depends on should list its name and location. For example:
-
-```yaml
-depends:
- - name: linux-baseline
-   url: https://github.com/dev-sec/linux-baseline/archive/master.tar.gz
- - name: ssh-baseline
-   url: https://github.com/dev-sec/ssh-baseline/archive/master.tar.gz
+```mermaid
```

Comment:
For some reason, this doesn't show up in the netlify deployment.

Reply:
Yeah I'm not sure why it doesn't show up. This is what it is supposed to look like. If it doesn't show up after merging, I can just remove it or use another way to design the flow chart. Thanks!

![Screen Shot 2023-02-07 at 11 32 04 AM](https://user-images.githubusercontent.com/11844975/217305131-029d85d2-587f-4081-9771-5a94876cc736.png)